### ISSUE TITLE
The QuickStart steps are impossible to follow

### ISSUE THREAD

**Issue by pballester** [angular/universal#673]
I tried to follow the quickstart but I found some major problems:

### Step 1
- It would me much better if a "git clone" command is provided
### Step 2
- No information provided about typings client installation
- The node typing doesn't exist anymore
- The ambient flag is deprecated
### Step 3
- The file name is server.js but it's typescript code  (#669)
- No information provided about compiling the file

Where is the git repository of the web page https://universal.angular.io/? I can submit a pull request if it's possible.

Thanks!

**Comment by albandaft** [comment]
Is there any way to make this RUN? Is there any documentation ?!!! So what is this just a useless repo? I have been searching all day and cannot find a tutorial or any steps to make universal/server side rendering working... seems all smoke and no fish

**Comment by fdambrosio** [comment]
We're unable to run Universal !

**Comment by fulls1z3** [comment]
I think the Angular team is currently developing/updating documentation for Universal. Meanwhile, better have a look at [ng-universal-demo](https://github.com/FrozenPandaz/ng-universal-demo).

I personally created a repo too, an example app showcasing both Universal and lean Angular on a single project [universal-example-app](https://github.com/nglibs/universal-example-app). But honestly I made it step-by-step following the instructions on [ng-universal-demo](https://github.com/FrozenPandaz/ng-universal-demo).

Again @FrozenPandaz, great job 👍

**Comment by MarkPieszak** [comment]
`angular2-universal` has since moved to `@angular/platform-server` within Angular Core.

You can check out the upgrade guide we have here as well: https://github.com/angular/universal/blob/master/UPGRADE-GUIDE.md

More complete documentation on Angular universal/platform-server/pre-rendering/patrickjs'ing coming soon!

**Comment by frasen** [comment]
@MarkPieszak For us who want to implement universal from scratch that is a bit hard to follow :) Any details on what "soon" means; e.g. do you think it is days, weeks or months?

**Comment by MarkPieszak** [comment]
@wardbell @vikerman might be able to answer that better. 
I know docs have been in the works for a bit, so you definitely won't have to wait months!

**Comment by frasen** [comment]
That's good. I would love something simple like https://universal.angular.io/quickstart/ for Angular 4! Let me know if I can help in any way.

**Comment by albandaft** [comment]
I spent days to make it work on Angular 4. Finally I wrote an article step by step how to make it work https://medium.com/@albanx/angular-server-side-rendering-5788d7b3c59c

**Comment by fulls1z3** [comment]
Also forgot to mention the article I've written a month ago about use of Angular Universal:
https://medium.com/burak-tasci/angular-4-with-server-side-rendering-aka-angular-universal-f6c228ded8b0

**Comment by frasen** [comment]
Loved the effort @albandaft but I got stuck since there seems to be quite a few implicit dependencies. Most likely because not very experienced in webpack.
